lib/rubocop/cop/layout/space_before_first_arg.rb

```diff
@@ -54,13 +54,21 @@ def regular_method_call_with_arguments?(node)
 
         def expect_params_after_method_name?(node)
           return false if node.parenthesized?
+          return true if touch_between_method_name_and_first_argument?(node)
 
           first_arg = node.first_argument
 
           same_line?(first_arg, node) &&
             !(allow_for_alignment? &&
               aligned_with_something?(first_arg.source_range))
         end
+
+        def touch_between_method_name_and_first_argument?(node)
```

Comment:
I'm not sure I understand the name of this method. What is touch between supposed to mean?

Reply:
Ah, I did a confusing naming 💦 
I'd like to change the name to `def no_space_between_method_name_and_first_argument?`

```ruby
# no space (bad)
do_something'arg' 

# with space (good)
do_something 'arg' 
```

Reply:
👍

Reply:
Thank you for your review. I updated the method name.